civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -7023,6 +7023,12 @@ function _block_anonymous_access() {
     'user/login',
   ];
 
+  // If we are not in maintenance mode then the guest users should
+  // be able to access vacancy apply public page.
+  if (variable_get('site_offline', 1)) {
+    $allowed_paths = array_merge(['civicrm/vacancy/apply'], $allowed_paths);
```

Comment:
actually, is this condition correct? Since they should be allowed to access this only if not in maintenance mode, shouldn't it be:
```php
if(variable_get('site_offline', 0)) {
}
```

Sorry if I'm wrong, but I don't know what is the value stored in site_offline for when it's in maintenance mode. I'm assuming it 1 in that case